TEST_PLAN.md

```diff
@@ -0,0 +1,13 @@
+# Test Plan
+
+## Pre-release
+1. Does the build pass on the CI server?
+1. On a fresh clone of the repository, do the example apps compile and send error reports?
+1. Has the changelog and any documentation been updated?
+1. Have all the version numbers been incremented?
+1. Has a release been created for this version on Github?
+
+## Post-release
+1. Have all Docs PRs been merged?
+1. Do the installation instructions work when creating an example app from scratch?
```

Comment:
Should these last two be a pre-release test?

Reply:
Yeah. Everything should work before release, and the released version should be tested after imo.